README.md

```diff
@@ -24,31 +24,68 @@ Create a new temporary project:
 * With a dependency that have a fixed version:
     `$ cargo-temp anyhow==1.0.13`
 
+### Repositories
+
+You can add repositories to you Cargo.toml.
+
+Examples:
+
+* HTTP
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git`
+
+* SSH
+    `$ cargo-temp anyhow=ssh://git@github.com/dtolnay/anyhow.git`
+
+This will add the repository on the default branch by default. You can choose
+another branch or a revision:
+
+* Branch
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#branch=main`
+
+* Revision
+    `$ cargo-temp anyhow=https://github.com/dtolnay/anyhow.git#rev=7e0f77a38`
+
 ## Features
 
-If you change your mind and decide to keep the project you can just delete the `TO_DELETE` file and the directory
-will not be deleted when the shell exits.
+If you change your mind and decide to keep the project you can just delete the
+`TO_DELETE` file and the directory will not be deleted when the shell exits.
```

Comment:
Is there a handy command we can do (like rm to_delete, or whatever) if so it could be included here.
Else if we have to do it by the gui, nevermind this comment

Reply:
You can certainly do `rm TO_DELETE` or remove it through your GUI. I expect devs know how to delete a file though :thinking:

Reply:
maybe i can do a two gif on how to delete the TO_DELETE file for both ?

Reply:
Not sure if you're joking or not

Reply:
Can we have gif support on PR comments please?!

Reply:
Maybe provide one GIF per IDE and editor? See you in 3 months lol